Minor updates to content sharing article (#4961)
SumoLogic/sumologic-documentation: docs/manage/content-sharing/index.md

import useBaseUrl from '@docusaurus/useBaseUrl';

> Content Sharing allows you to selectively share and collaborate on apps, dashboards, and searches with specific users or roles. As an Admin, you can use content sharing to share ownership of searches and dashboards, or to highlight key content to specific users and groups. As a user, you can now choose how widely shared your content is within your organization.

Reviewer: ```suggestion
Content Sharing allows you to selectively share and collaborate on apps, dashboards, and searches with specific users or roles. As an Admin, you can use content sharing to share ownership of searches and dashboards, or to highlight key content to specific users and groups. As a user, you have the flexibility to control how broadly your content is shared within your organization.
```
